doc/helpers/FlipView-extensions.md

```diff
@@ -20,32 +20,43 @@ The `Next` and `Previous` properties will provide an easy hook to allow other co
 <!-- Include the following XAML namespace to use the samples below -->
 xmlns:utu="using:Uno.Toolkit.UI"
 ...
-
 <Grid>
-    <FlipView>
-        <FlipView.Items>
-            <Grid Background="Azure">
-                <Button
-                    HorizontalAlignment="Left"
-                    utu:FlipViewExtensions.Previous="{Binding ElementName=flipView}"
-                    Content="Previous" />
-                <Button
-                    HorizontalAlignment="Right"
-                    utu:FlipViewExtensions.Next="{Binding ElementName=flipView}"
-                    Content="Next" />
-            </Grid>
-            <Grid Background="Blue">
-                <Button
-                    HorizontalAlignment="Left"
-                    utu:FlipViewExtensions.Previous="{Binding ElementName=flipView}"
-                    Content="Previous" />
-                <Button
-                    HorizontalAlignment="Right"
-                    utu:FlipViewExtensions.Next="{Binding ElementName=flipView}"
-                    Content="Next" />
-            </Grid>
-        </FlipView.Items>
-    </FlipView>
+  <Grid.RowDefinitions>
+    <RowDefinition Height="*" />
+    <RowDefinition Height="Auto" />
+  </Grid.RowDefinitions>
+
+  <FlipView x:Name="flipView">
+    <FlipView.Items>
+      <Grid Background="Tomato">
+        <TextBlock Text="Item 1"
+                   HorizontalAlignment="Center"
+                   VerticalAlignment="Center" />
+      </Grid>
+      <Grid Background="CornflowerBlue">
+        <TextBlock Text="Item 2"
+                   HorizontalAlignment="Center"
+                   VerticalAlignment="Center" />
+      </Grid>
+      <Grid Background="Goldenrod">
+        <TextBlock Text="Item 3"
+                   HorizontalAlignment="Center"
+                   VerticalAlignment="Center" />
+      </Grid>
+    </FlipView.Items>
+  </FlipView>
+
+  <StackPanel Orientation="Horizontal"
+              HorizontalAlignment="Center"
+              Grid.Row="1">
+    <Button HorizontalAlignment="Left"
+            utu:FlipViewExtensions.Previous="{Binding ElementName=flipView}"
+            Content="Previous" />
+    <Button HorizontalAlignment="Right"
+            utu:FlipViewExtensions.Next="{Binding ElementName=flipView}"
+            Content="Next" />
+  </StackPanel>
+</Grid>
 ```
 
 ## Customization
```

Comment:
There are also some typos in this section, can you proof-read the Customization section? Might as well fix it in this PR as well